src/components/Text/index.js

```diff
@@ -1,10 +1,16 @@
-import React from "react"
 
-const Text = (props) => {
-  const { text } = props
+import React from "react";
+import './Text.css';
 
+//<Text text='Sample text' color='green' size="small" />
```

Comment:
@AnthoniaOkafor, can you please remove all commented code from your PR?

Reply:
I will @meekg33k